packages/api-explorer/src/components/DocPseudo/DocPseudo.tsx

```diff
@@ -38,11 +40,13 @@ export const DocPseudo: FC<PseudoProps> = ({ method }) => {
   const resType = method.primaryResponse
 
   return (
-    <Flex alignItems="flex-start">
-      {`${method.operationId}(`}
-      <DocParams parameters={args} />
-      {'): '}
-      <DocPrimaryResponse response={resType} />
+    <Flex alignItems="flex-start" fontSize="small" fontWeight="normal">
+      <Code fontSize="small" fontWeight="normal">
```

Comment:
Double `fontSize` / `fontWeight` here seems odd. Is there a reason the `Flex` is needed at all?